studio/components/interfaces/Settings/General/Infrastructure/ProjectUpgradeAlert/ProjectUpgradeAlert.tsx

```diff
@@ -83,44 +87,89 @@ const ProjectUpgradeAlert = () => {
                         All services, including Auth, Rest, and Extensions will be upgraded. This
                         action cannot be undone.
```

Comment:
@pcnc what are you trying to say? I feel like this is saying 1 thing expecting the user to know something. I'm we are warning them that the instance is going to go offline?

Reply:
It refers to the service versions mentioned on the project's infrastructure page, as well as other extensions, with the intent of letting the user know there is no way to downgrade after they've upgraded their project.

![26938](https://github.com/supabase/supabase/assets/30039543/7bc63981-53d4-4f6f-a47d-05795240fa17)

Reply:
my 2 cents :D

Reply:
Sounds good! Will wrangle the changes into code 👀